electron/index.js

```diff
@@ -18,6 +18,7 @@ const electron = require('electron');
 const fs = require('fs');
 const path = require('path');
 const url = require('url');
+const { URL, URLSearchParams } = url;
```

Comment:
`const { format, URL, URLSearchParams} = require('url')`, or put this line under all the requires.

The linter doesn't yell here, but I think it's one of the rules to first import then const ..., so let's apply it here too.